### PR TITLE
UNO 235 - Move ckeditor to tao-core-shared-libs-fe

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/UNO-235

`CKeditor` was moved to https://github.com/oat-sa/tao-core-shared-libs-fe/pull/1
Updated README.md

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]

**Comment by habemuscode** [comment]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ cd dev/builder
 sudo php oneFile.php
 ```
 
-The release is located under `release/ckeditor-reduced`. The complete folder can be copied into `tao/views/js/lib/ckeditor`
+The release is located under `release/ckeditor-reduced`. The complete folder can be copied into into `lib/ckeditor/` of [`@oat-sa/tao-core-libs`](https://github.com/oat-sa/tao-core-libs-fe)
```
Comment (btamas) [inline README.md]: `can be copied into into`

File: README.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ The built versions checksums aren't identical because the source code contains t
     - `git checkout develop -- core/scriptloader.js`
 
 #### TAO Skin
-- If you need to modify TAO skin, you'll find the SASS source files in `tao/views/js/lib/ckeditor/skins/tao/scss`. They are not in this repo. 
-- To compile, use the grunt task `grunt sass:ckeditor`
-- Once compiled, copy the css files from `tao/views/js/lib/ckeditor/skins/tao/*` into the `skins/tao/` folder of this repository. This will avoid accidental override of the skin when moving a new CK build into TAO.
+- If you need to modify TAO skin, you'll find the SASS source files in [`@oat-sa/tao-core-ui-fe`](https://github.com/oat-sa/tao-core-ui-fe/tree/master/scss/ckeditor/skins/tao/scss). They are not in this repo. 
+- To compile, use the grunt task `npm run buildScss`
+- Once compiled, copy the css files from `tao-core-ui-fe/css/ckeditor/skins/tao/*` into the [`skins/tao/`](https://github.com/oat-sa/ckeditor-dev/tree/develop/skins/tao) folder folder of this repository. This will avoid accidental override of the skin when moving a new CK build into TAO.
```
Comment (btamas) [inline README.md]: `folder folder of this repository`

File: skins/tao/css/editor.css [hunk 3]
```diff
@@ -201,247 +45,92 @@ editors:
...
-  width: auto;
-  height: auto;
-  vertical-align: baseline;
-  box-sizing: content-box;
-  -moz-box-sizing: content-box;
-  -webkit-box-sizing: content-box;
-  position: static;
-  -webkit-transition: none;
-  -moz-transition: none;
-  -ms-transition: none;
```
Comment (btamas) [inline skins/tao/css/editor.css:385]: Are you sure the browserlist is correct? Box sizing is prefixed for webkit and moz, but not for ms?

**Review by btamas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bziondik) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (bziondik) [reply README.md]: fixed

File: skins/tao/css/editor.css (hunk 3 above)
Comment (bziondik) [reply skins/tao/css/editor.css:385]: Cleaned sourse scss from css-properties with prefix in this commit https://github.com/oat-sa/tao-core-ui-fe/pull/123/commits/e9293c08006bad95a19e70cf0af866eea4769d13
And postcss in oat-sa/tao-core-ui-fe use
"browserslist": [
        "extends @oat-sa/browserslist-config-tao"
    ]
https://github.com/oat-sa/tao-core-ui-fe/blob/d9c1eb9579efad06c2310fe3b981d8a5ec3f118a/package.json#L88

**Review by btamas — APPROVED** [review]

**Pull request merged**